packages/js/src/ui/context/LocalizationContext.tsx

```diff
@@ -1,9 +1,10 @@
 import { ParentProps, createContext, createMemo, useContext } from 'solid-js';
-import { defaultLocalization } from '../config/default-localization';
+import { defaultLocalization } from '../config/defaultLocalization';
 import { Path } from '../helpers/types';
 
 export type Localization = {
   'inbox.title': string;
+  'notifications.emptyNotice': string;
```

Comment:
@desiprisg to avoid adding a type key every time we add a new localization key, let's generate the type from the typeof the Localization object.